UI/Panels/Settings/MobiFlightPanel.cs

```diff
@@ -565,8 +565,31 @@ private void addDeviceTypeToolStripMenuItem_Click(object sender, EventArgs e)
                             throw new MaximumDeviceNumberReachedMobiFlightException(MobiFlightLcdDisplay.TYPE, tempModule.Board.ModuleLimits.MaxLcdI2C);
                         }
 
+                        // Check and see if any I2CPins exist in the board definition file. If not then there's no way to add an LCD device
+                        // so throw an error.
+                        var I2CPins = tempModule.Board.Pins.FindAll(x => x.isI2C);
+
+                        if (!(I2CPins?.Any() ?? false))
```

Comment:
This is a basic test for any I2C pins being defined in the board definition file. Technically I guess this could check for exactly two pins being defined as I2C but that seems soooooo edge case.

Reply:
The naming of the variables could be improved to reduce the required lines of //comments

e.g I2CPins => AvailableI2Cpins

pinsInUse is clearly named.

inUseI2CPin => improve reading by naming it what the if test is doing: "IsAnyI2CPinAlreadyInUse"

Reply:
Renamed `I2CPins` to `availableI2CPins`.

Renamed `inUseI2CPin` to `firstInUseI2CPin`. It doesn't wind up as a boolean, it winds up as the actual pin, so naming it `IsAnyI2CPinAlreadyInUse` doesn't make sense.